src/parquet/util/rle-test.cc

```diff
@@ -218,13 +218,13 @@ bool CheckRoundTrip(const vector<int>& values, int bit_width) {
       return false;
     }
   }
-  int encoded_len = encoder.Flush();
+  encoder.Flush();
```

Comment:
The encoded length is data-dependent, but I set it to initialize the decoder with the encoded length